advanced/Scripts/webpage.sh

```diff
@@ -557,23 +557,27 @@ addAudit()
     shift # skip "-a"
     shift # skip "audit"
     local domains validDomain
-    domains="('$(checkDomain "${1}")')"
-    shift # skip first domain, as it has already been added
+    domains=""
     for domain in "$@"
     do
       # Insert only the domain here. The date_added field will be
       # filled with its default value (date_added = current timestamp)
       validDomain="$(checkDomain "${domain}")"
       if [[ -n "${validDomain}" ]]; then
-        domains="${domains},('${domain}')"
+        # Put comma in between () when there is
```

Comment:
What does `()` represent?

Reply:
It forms a single set of variables. With multiple sets of variables, we can add multiple rows at once.

Adding one row using SQL insertion (as usual):
```sql
INSERT INTO table (domain) VALUES ('abc.de');
```

Adding multiple rows with one insertion:
```sql
INSERT INTO table (domain) VALUES ('abc.de'),('fgh.ij'),('klm.no'),('pqr.st');
```

Reply:
Ah, the comment was confusing (sounded like it said `(,)`). Maybe instead say "Put comma in between domains"?